src/wp-includes/block-patterns.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Register the block patterns and block patterns categories
+ *
+ * @package WordPress
+ * @since 5.5.0
+ */
+
+/**
+ * Loads a Core block pattern.
+ *
+ * @since 5.5.0
+ * @access private
+ *
+ * @param string $name Block pattern name.
+ * @return array Block pattern settings.
+ */
+function _load_block_pattern( $name ) {
```

Comment:
Should we minimise the risk of naming collision by inlining the `require` calls?

Reply:
Can you clarify what naming collision you're thinking of?

Reply:
Not a big deal, but it's noticeable how such a local helper function ends up polluting the global function namespace.

I guess you could also use an anonymous function ([PHP 5.3](https://www.php.net/manual/en/functions.anonymous.php)), or avoid functions entirely with a for loop looping over the pattern slugs.

Reply:
I like the loop proposal, I'll update the code.

Reply:
Updated.

Reply:
Cool. Commit! 🚀